tox.ini

```diff
@@ -0,0 +1,137 @@
+[tox]
+requires =
+    tox>=4
+env_list = lint
+
+[style]
+description = common environment for style checkers
+skip_install = true
+deps =
+    isort
+    black
+    blacken-docs
+    flake8
+    flake8-docstrings
+    flake8-use-fstring
+    flake8-functions
+    codespell
+    tomli
+
+[test]
+description = base environment for testing
+skip_install = false
+deps =
+    coverage
+    pytest>=6.0.0
+    pytest-cov
+
+[min]
+description = environment with minimum versions of all dependencies (plotting not included)
+skip_install = false
+deps =
+    joblib==1.0.0
+    nibabel==3.2.0
+    numpy==1.19.0
+    pandas==1.1.5
+    scikit-learn==1.0.0
+    scipy==1.6.0
+
+[plotmin]
+description = environment with minimum matplotlib version
+skip_install = false
+deps =
+    matplotlib==3.3.0
+
+[matplotlib]
+description = environment with matplotlib dependencies
+skip_install = false
+deps =
+    matplotlib>=3.3.0
+
+[plotly]
+description = environment with plotly dependencies
+skip_install = false
+deps =
+    {[matplotlib]deps}
```

Comment:
should plotly contain mpl dependency? if so for the way you've set it up maybe change the name of this section to plotting

Reply:
makes sense